src/main/java/com/example/bookshop/controller/BookController.java

```diff
@@ -29,31 +30,36 @@ public class BookController {
 
     @Operation(summary = "Get all books", description = "Get a list with all books")
     @GetMapping
+    @PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_USER')")
```

Comment:
Since users might hold various roles, I'm not entirely sure if it's essential to specify 'ROLE_ADMIN'. 😊
```suggestion
    @PreAuthorize("hasRole('ROLE_USER')")
```

Reply:
Agree